spring-cloud-gcp-data-firestore/src/test/java/com/google/cloud/spring/data/firestore/entities/UserRepository.java

```diff
@@ -36,6 +36,10 @@
 
 	Flux<User> findByAge(Integer age);
 
+	Flux<User> findByAge(Integer age, Sort sort);
+
+	Flux<User> findByAgeOrderByNameDesc(Integer age);
```

Comment:
Since `name` or document id is a somewhat special field in Firestore, can we also add a test that sorts by a regular field like `age`? Is something like `findAllOrderByAge()` possible?